funnel/templates/index.html.jinja2

```diff
@@ -1,14 +1,21 @@
 {% extends "layout.html.jinja2" %}
 {% block title %}{% if profile %}{{ profile.title }}{% else %}{{ config['SITE_TITLE'] }}{% endif %}{% endblock %}
 
-{% block description %}{{ config['WELCOME_MESSAGE'] }}{% endblock %}
+{% block description %}{% if profile %}{{ profile.description.html }}{% else %}{{ config['WELCOME_MESSAGE'] }}{% endif %}{% endblock %}
```

Comment:
Welcome message needs to be removed from config and added directly to the template.